README.md

```diff
@@ -112,19 +112,25 @@ $ ./packages/percolatestudio-migrations/migrate.sh latest --settings ./setting.j
 
   Migrations set a lock when they are migrating, to prevent multiple instances to prevent multiple instances of your clustered app from running migrations simultaneously. If you migrations throw an exception, you will need to manually remove the lock (and ensure your db is still consistent) before re-running the migration. Update the migrations collection like this:
 
-  ``` sh
-  meteor mongo
+  ```
```

Comment:
I chose not to remove the sh language indicator from this fenced code block as only the first line is actually a true shell command.